test/index.test.ts

```diff
@@ -55,3 +58,21 @@ describe.each(allTypoSets)('%s', (name, data) => {
 		expect(contextDisplays, name).toMatchObject(data.displays);
 	});
 });
+
+describe('findConfig', () => {
+	test('should find a mock config file in working directory', async () => {
+		const readFileSpy = vi.spyOn<typeof fs, 'readFile'>(fs, 'readFile');
+
+		readFileSpy.mockImplementation(((path: string, _encoding: any, callback: (err: null, data: string) => void) => {
+			if (path === join(process.cwd(), 'cspell.json')) {
```

Comment:
Since `join(process.cwd(), 'cspell.json')` is used twice, create a `configPath` variable.